fluent/sender.py

```diff
@@ -122,6 +122,13 @@ def close(self):
             self._close()
             self.pendings = None
 
+    def _is_ipv6_host(self):
+        try:
+            socket.getaddrinfo(self.host, None, socket.AF_INET6)
```

Comment:
This will result in host that has AAAA records to prefer AF_INET6 even if AF_INET IPv4 addresses are available. This is not the current behavior and will potentially break tons of stuff for people who are not expecting this behavior.